### ISSUE TITLE
Weird errors after latest plugin update (on July 13, 2022)

### ISSUE THREAD

**Issue by krishnakumarg1984** [gpanders/editorconfig.nvim#15]
I get the following errors when opening any file e.g. `init.lua` in neovim.

```
editorconfig: invalid value for option end_of_line: lf. ...pack/packer/start/editorconfig.nvim/lua/editorconfig.lua:22: Cannot unset option 'fileformat' because it doesn't have a global value
editorconfig: invalid value for option indent_size: 2. ...pack/packer/start/editorconfig.nvim/lua/editorconfig.lua:43: Cannot unset option 'shiftwidth' because it doesn't have a global value
editorconfig: invalid value for option end_of_line: lf. ...pack/packer/start/editorconfig.nvim/lua/editorconfig.lua:22: Cannot unset option 'fileformat' because it doesn't have a global value
editorconfig: invalid value for option indent_size: 2. ...pack/packer/start/editorconfig.nvim/lua/editorconfig.lua:43: Cannot unset option 'shiftwidth' because it doesn't have a global value
```

This is a new issue i.e. I never had this issue before ever since I installed `editorconfig.nvim`. My global editorconfig config file `~/.editorconfig` is available [here](https://gist.github.com/krishnakumarg1984/22e52dd88e8561b0ab3b3920ac602c14)

**Comment by gpanders** [comment]
What is the output of `:version` in Neovim?

**Comment by gpanders** [comment]
Nevermind, I found the bug. Thanks for reporting!

**Comment by gpanders** [comment]
Fixed in 918279f.

**Comment by krishnakumarg1984** [comment]
Yup works now. Thanks for fixing this so quickly. JFYI, my `:version` output is:

```
NVIM v0.7.2
Build type: Release
LuaJIT 2.1.0-beta3
Compiled by linuxbrew@649894113f22

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/home/linuxbrew/.linuxbrew/Cellar/neovim/0.7.2/share/nvim"

Run :checkhealth for more info
```
